README.md

```diff
@@ -1,11 +1,16 @@
-uMirrorMaker provides the ability to replicate between Kafka clusters in other data centers. With uMirrorMaker, you can publish to multiple regional Kafka clusters and aggregate in an aggregation Kafka cluster instead of publishing to a single Kafka cluster.
-The current MirrorMaker design uses the Kafka high level consumer to consume data from multiple regional kafka clusters and produces to an aggregation kafka cluster. With this design, whenever a regional kafka broker is having an issue, it will cause rebalancing to happen across all of the active topics.
+uMirrorMaker 
+============
+uMirrorMaker provides the ability to replicate across Kafka clusters in other data centers. Instead of publishing to a single Kafka cluster, you can publish data to multiple regional Kafka clusters and aggregate it all in one Kafka cluster.
 
-uMirrorMaker introduces a helix controller as the MirrorMaker Controller and uses a simple consumer in MirrorMaker Worker. This design achieves the following goals:
+=========
 
-1. Stable MM: Rebalancing only during startup (when a node is added/ deleted)
-2. Simple operations: easy to scale up cluster, no server restart for whitelisting topics
-3. High throughput: max offset lag is consistently 0.
+Kafka's current (part of 0.8.2) MirrorMaker design consumes data from multiple regional Kafka clusters using a Kafka high-level consumer. With this design, any regional Kafka broker experiencing issues causes rebalancing across all of the active topics.
```

Comment:
I think we got this wrong when we first wrote this. It shoudl be : 

Kafka's current (part of 0.8.2) MirrorMaker design consumes data from "a given regional Kafka cluster" using a Kafka high-level consumer ....